src/TextLink/TextLink.js.flow

```diff
@@ -7,8 +7,8 @@ export type Props = {|
   children: React$Node,
   href?: string,
   onClick?: (SyntheticEvent<HTMLLinkElement>) => void,
-  external: boolean,
-  type: Type,
+  external?: boolean,
```

Comment:
I also had to change `external` and `type` to be optional. Can be merged in this PR or we should create another one.

Reply:
It is OK to be merged with this 💪